Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -1449,3 +1461,22 @@ theorem hasFPowerSeriesAt_iff' :
 #align has_fpower_series_at_iff' hasFPowerSeriesAt_iff'
 
 end
+
+/-!
+### Combining analytic functions: sums, products, and pairs
+-/
+
+/-- Finite sums of analytic functions are analytic -/
+theorem AnalyticAt.sum {f : ℕ → E → F} {c : E} (h : ∀ n, AnalyticAt 𝕜 (f n) c) (N : Finset ℕ) :
+    AnalyticAt 𝕜 (fun z ↦ N.sum fun n ↦ f n z) c := by
+  induction' N using Finset.induction with a B aB hB
+  · simp only [Finset.sum_empty]; exact analyticAt_const
+  · simp_rw [Finset.sum_insert aB]
+    apply AnalyticAt.add
+    exact h a
+    exact hB
```

Comment:
Not completely essential, but I think it's good style to use `· ` every time there are multiple goals.

Reply:
Is there a way to lint for this?

Reply:
I think it would be tricky to lint for. There are valid tactics to run when you have multiple goals open (`rotate_right`, `solve_by_elim*`, ...).